crates/miden-lib/build.rs

```diff
@@ -26,7 +26,7 @@ type ErrorCategoryMap = BTreeMap<ErrorCategory, Vec<NamedError>>;
 /// Defines whether the build script should generate files in `/src`.
 /// The docs.rs build pipeline has a read-only filesystem, so we have to avoid writing to `src`,
 /// otherwise the docs will fail to build there. Note that writing to `OUT_DIR` is fine.
-const BUILD_GENERATED_FILES_IN_SRC: bool = option_env!("BUILD_GENERATED_FILES_IN_SRC").is_some();
+const BUILD_GENERATED_FILES_IN_SRC: bool = true; // option_env!("BUILD_GENERATED_FILES_IN_SRC").is_some();
```

Comment:
Probably a leftover from development?

I tend to run `make build-no-std` to trigger rebuild of generated files, maybe this helps :)

Reply:
Woops, yep, I just forgot to roll back this change. Yes, for some reason the rebuilding the generated files works unreliable for me, I'll try to use the `make build-no-std` as you suggested. Thank you!